update.js

```diff
@@ -34,6 +34,10 @@ module.exports = function update () {
     return console.error('This build should not update the lockfile. It could be a PR, not a branch build.')
   }
 
+  if (info.branchName == null) {
```

Comment:
My thinking was that `''` would at least work with `.startsWith()` later down the code, and I didn't want to introduce any new behaviour.

But your suggestion looks good. Happy to make the change if you think it makes sense.